README.md

```diff
@@ -147,37 +179,87 @@ container, not the host, e.g.,
 bam:
   class: File
   path: /data/sample.bam
-ref_db_prefix: /references/ref.fa
+ref_db_prefix: /reference/ref.fa
 ```
 
-The following is an example `run` command where files are stored in `test/{data,references}`. Outputs are saved in `test/results`.
+The following is an example `run` command where files are stored in `test/{data,reference}`. Outputs are saved in `test/results`.
 
-This example assumes you are running against Mus Musculus (genome build MGSCv37). Set the path to the folder containing your reference data
+This example assumes you are running against Mus musculus (genome build MGSCv37). Set the path to the folder containing your reference data
 and run the following command to produce output from the included sample data. Test output for comparison is located at `sample_data/output_data`.
 
 ```
 $ mkdir $(pwd)/results
 $ docker run \
   --mount type=bind,source=$(pwd)/sample_data/input_data,target=/data,readonly \
-  --mount type=bind,source=/path/to/references,target=/references,readonly \
+  --mount type=bind,source=/path/to/reference,target=/reference,readonly \
   --mount type=bind,source=$(pwd)/results,target=/results \
   xenocp \
   /data/inputs.yml
 ```
 
 [Dockerfile]: ./Dockerfile
 
+## Evaluate test data results
+
+If you have [bcftools] and a [GRCh37-lite] reference file, the following will show two variants in the input file. The variant on chromosome 1 is a variant in the host genome. The variant on chromosome 9 is a variant in the graft genome. 
```

Comment:
Should probably break across lines for source readability.

Also, I'm assuming the variant on chromosome 1 is not really a variant in the host genome, but probably a non-variant in the host genome that appears to be a variant when mapped against the graft genome.  So, perhaps say "The variant on chromosome 1 is an artifact of mouse reads.  The variant on chromosome 9..."